### PR TITLE
validator: Transfer validator logic to seperate file

### PR DESCRIPTION
**Pull request by dhruvja**
Validator logic moved to a seperate file since it is called during a single command only. Renamed `parse_config` with `parse_command` which gets the required arguments and config and calls the method to execute. Using this we can add more commands with its own logic.

### CODE REVIEW

**Review by blasrodri — COMMENTED** [review]
Please make the comments more verbose, and this will be executed by third parties.

File: validator/src/validator.rs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+use anchor_client::{Client, Cluster};
+use anchor_lang::solana_program::pubkey::Pubkey;
+use lib::hash::CryptoHash;
+use solana_ibc::{accounts, instruction};
+
+use crate::command::Config;
+use crate::utils;
+
+pub fn run_validator(config: Config) {
+    let validator = Rc::new(Keypair::from(config.keypair));
```
[the remark this answers is not in the document]
Comment (dhruvja) [reply validator/src/validator.rs:21]: Well yes. It will only fail if the config is wrong. I am unwrapping but better error handling is required for sure.

**Comment by dhruvja** [comment]
> Please make the comments more verbose, and this will be executed by third parties.

sure sure will do.

File: validator/src/validator.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (blasrodri) [reply validator/src/validator.rs:21]: mostly because this will be ran by external people

**Pull request merged**